buildSrc/src/main/resources/deb/postinst.ftl

```diff
@@ -1,2 +1,3 @@
-#!/usr/bin/env bash -e
+#!/usr/bin/env bash
+set -e
```

Comment:
Can we set it something like `set -e -o pipefail`? Just to maintain the consistency around the scripts as did here: [set -e -o pipefail](https://github.com/opensearch-project/OpenSearch/pull/3278)

Reply:
+1 for consistency

Update pushed incorporating this change.